### PR TITLE
Docs-4235 SCALE 22.02 Update S3 Article Content

### PR DESCRIPTION
**Pull request by micjohnson777**
This commit updates the S3 UI ref and how-to article content.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALETutorials/SystemSettings/Services/S3ServiceSCALE.md [hunk 1]
```diff
@@ -27,29 +28,24 @@ Because S3 is the de facto standard for cloud-based storage, setting up an S3 se
...
 
-{{< expand "Field Descriptions" "v" >}}
-{{< include file="/_includes/ServicesS3Fields.md" type="page" >}}
-{{< /expand >}}
+![S3ServiceSettings(/images/SCALE/22.02/S3ServiceSettings.png "S3 Service Options")
 
-Select a clean dataset, one that doesn't have existing data files.
-Minio manages files as objects that you *cannot* mix with other dataset files.
-You can create new datasets by going to **Storage** and clicking <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> > **Add Dataset**.
+First, select a clean dataset, one that does not have existing data files. If you do not have a clean dataset, [create a dataset]({{< relref "DatasetSCALE.md" >}}).
```
Comment (Mrt134) [inline content/SCALE/SCALETutorials/SystemSettings/Services/S3ServiceSCALE.md:35]: Build test found an error (**Page not found**) with this reference:
image

File: content/SCALE/SCALEUIReference/SystemSettings/Services/S3ServiceScreen.md [hunk 2]
```diff
@@ -13,98 +13,20 @@ tags:
...
+| **Port** | Enter the TCP port that provides the S3 service.   |
+| **Console Port** | Enter a static port for the MinIO web console. Default is 9001. |
+| **Access Key** | Enter the S3 access ID. See [Access keys](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) for more information. |
+| **Secret Key** | Enter the S3 secret access key. See [Access keys](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) for more information. |
+| **Disk** | Enter or use <iconify-icon icon="bxs:right-arrow"></iconify-icon> to the left of <span class="material-icons">folder</span> **/mnt** to browse to a directory to define the S3 file system path. |
+| **Enable Browser** | Enables the S3 service web UI. Access the MinIO web UI by entering the IP address and port number separated by a colon in the browser address bar. Example: *192.168.1.0:9000*. |
+| **Certificate** | Use an SSL [certificate]({{< relref "CertificatesSCALE.md" >}}) created or imported in **Credentials > Certificates** for secure S3 connections. |
```
Comment (Mrt134) [inline content/SCALE/SCALEUIReference/SystemSettings/Services/S3ServiceScreen.md:29]: Build test found this reference is ambiguous. It'll need to be adjusted with the path to the desired file.
image

```suggestion
| **Certificate** | Use an SSL [certificate]({{< relref "CertificatesSCALE.md" >}}) created or imported in **Credentials > Certificates** for secure S3 connections. |
```

**Review by Mrt134 — CHANGES_REQUESTED** [review]
Build test found two errors that need to be fixed.

File: content/SCALE/SCALETutorials/SystemSettings/Services/S3ServiceSCALE.md (hunk 1 above)
Comment (micjohnson777) [reply content/SCALE/SCALETutorials/SystemSettings/Services/S3ServiceSCALE.md]: fixed link with a typo in the file name, datasetsSCALE.md is the correct file name.

**Pull request merged**